.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,31 @@
+---
+name: Bug report
+about: Create a report to help us improve
+labels: bug
+---
+
+<!-- Please search existing issues to avoid creating duplicates. -->
+
+
+#### Code Sample, a copy-pastable example if possible
+
+A "Minimal, Complete and Verifiable Example" will make it much easier for maintainers to help you:
+http://matthewrocklin.com/blog/work/2018/02/28/minimal-bug-reports
+
+```c
+# Your code here
+
+```
+#### Problem description
+
+[this should explain **why** the current behavior is a problem and why the expected output is a better solution.]
+
+#### Expected Output
+
+
+#### Environment Information
+ - PROJ version (`proj`)
+ - Operation System Information (`python -c "import platform; print(platform.platform())"`)
```

Comment:
Python can't be expected to be present on Windows - better leave platform agnostic, people will know what to do.
```suggestion
 - Operation System Information
```